test/utils-tests.jl

```diff
@@ -468,6 +468,15 @@ let
     @test obl  ≈ [-9.507794266102866, -6.970768250588256]
 end
 
+# Test obliquity
+@testset "obliquity" begin
+    @test obliquity(2.451545e6) ≈ 0.4090646078966446
+    eps_out = obliquity.(jdcnv.([DateTime(2016, 08, 23, 03, 39, 06),
```

Comment:
`isapprox` doesn't work with tuples (that's why many functions here have to assign the result to a named variable for testing purposes), but does work with arrays, so you can simply do
```julia
@ obliquity.(jdcnv.([DateTime(2016, 08, 23, 03, 39, 06), DateTime(763, 09, 18, 12)])) ≈
    [..., ...]
```